### ISSUE TITLE
The testcase test_graph_unit_dnnl_layer_norm_usm_cpu fails

### ISSUE THREAD

**Issue by yurivict** [oneapi-src/oneDNN#2011]
# Summary

```
----------------------------------------------------------

113/125 Testing: test_graph_unit_dnnl_layer_norm_usm_cpu
113/125 Test: test_graph_unit_dnnl_layer_norm_usm_cpu
Command: "/usr/ports/math/onednn/work/.build/tests/gtests/graph/unit/test_graph_unit" "--gtest_filter=test_layer_norm_*" "--engine=cpu"
Directory: /usr/ports/math/onednn/work/.build/tests/gtests/graph/unit
"test_graph_unit_dnnl_layer_norm_usm_cpu" start time: Jul 27 00:19 PDT
Output:
----------------------------------------------------------
Note: Google Test filter = test_layer_norm_*:-*_GPU*
[==========] Running 6 tests from 2 test suites.
[----------] Global test environment set-up.
[----------] 5 tests from test_layer_norm_execute
[ RUN      ] test_layer_norm_execute.LayernormNDimCheck
<end of output>
Test time =   0.17 sec
----------------------------------------------------------
Test Failed. 
"test_graph_unit_dnnl_layer_norm_usm_cpu" end time: Jul 27 00:19 PDT
"test_graph_unit_dnnl_layer_norm_usm_cpu" time elapsed: 00:00:00
----------------------------------------------------------
```

# Version
Report oneDNN version and githash. Version information is printed to stdout
in [verbose mode](https://oneapi-src.github.io/oneDNN/dev_guide_verbose.html).

# Environment
* CPU make and model (try `lscpu`; if your `lscpu` does not list CPU flags,
  try running `cat /proc/cpuinfo | grep flags | sort -u`)
* OS version FreeBSD 14.1
* Compiler version clang-18
* Version: 3.5.2

**Comment by TaoLv** [comment]
@rongzha1 Could you please take a look on this?

**Comment by rongzha1** [comment]
thanks for your report, could you show me the build log?
I can't reproduce it in my env.

**Comment by yurivict** [comment]
[here](https://freebsd.org/~yuri/onednn-3.5.2-test.log) is the full log.

**Comment by yurivict** [comment]
cmake args:
-DDNNL_CPU_RUNTIME=OMP -DCMAKE_C_COMPILER:STRING="cc"  -DCMAKE_CXX_COMPILER:STRING="c++"  -DCMAKE_C_FLAGS:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing "  -DCMAKE_C_FLAGS_DEBUG:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing "  -DCMAKE_C_FLAGS_RELEASE:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -DNDEBUG"  -DCMAKE_CXX_FLAGS:STRING="-O3 -pipe   -isystem /usr/local/include "  -DCMAKE_CXX_FLAGS_DEBUG:STRING="-O3 -pipe   -isystem /usr/local/include "  -DCMAKE_CXX_FLAGS_RELEASE:STRING="-O3 -pipe   -isystem /usr/local/include  -DNDEBUG"  -DCMAKE_EXE_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_MODULE_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_SHARED_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_INSTALL_PREFIX:PATH="/usr/local"  -DCMAKE_AUTOGEN_PARALLEL:STRING="7"  -DCMAKE_BUILD_TYPE:STRING="Release"  -DTHREADS_HAVE_PTHREAD_ARG:BOOL=YES  -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=YES  -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON  -DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=ON -DDNNL_BUILD_TESTS:BOOL=OFF -DDNNL_BUILD_EXAMPLES:BOOL=OFF -GNinja

**Comment by TaoLv** [comment]
Hi @yurivict, recently we fixed a bug in the unit test (d815a479118d884e65151be79f5abc1bdd52146c). Could you please check if you still see this issue in your environment? Thanks.

**Comment by vpirogov** [comment]
Fixed by 684a6a4314ed81849d99b1f60469354dcccd784a.
